scripts/data.py

```diff
@@ -1,7 +1,12 @@
+from pathlib import Path
+
+FOLDER = Path(__file__).parent
```

Comment:
I'd change `FOLDER` to `SRC_DIR` -- assuming that `scripts` changes to `src`, which is more standard. Otherwise, `FOLDER` can be replaced with `SCRIPTS_DIR`. `FOLDER` itself is too ambiguous.

But good change to improve path handling.